packages/mds-web-sockets/clients.ts

```diff
@@ -49,7 +50,9 @@ export class Clients {
       trimmedEntities.map(entity => {
         try {
           this.subList[entity].push(client)
+          client.send(`SUB%${JSON.stringify({ status: 'Success' })}`)
```

Comment:
This was an opportunistic add in order to write an integration test to confirm subscription success.  Let me know if there's potential for this to cause any breakage downstream